Application/EdFi.Ods.Api/Filters/EnforceAssignedProfileUsageFilter.cs

```diff
@@ -97,7 +97,7 @@ public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionE
                         ? contentTypes.Readable
                         : contentTypes.Writable)
                     != null)
-            .ToArray();
+            .OrderBy(a=>a).ToArray();
```

Comment:
Why does this need to be sorted?

Reply:
So that string errorMessage will be same for both database

Reply:
string errorMessage = relevantContentTypeUsage == ContentTypeUsage.Readable
                ? $"Based on profile assignments, one of the following profile-specific content types is required when requesting this resource: '{string.Join("', '", assignedProfilesForRequest.Select(p => ProfilesContentTypeHelper.CreateContentType(resourceFullName.Name, p, relevantContentTypeUsage)))}'"
                : $"Based on profile assignments, one of the following profile-specific content types is required when updating this resource: '{string.Join("', '", assignedProfilesForRequest.Select(p => ProfilesContentTypeHelper.CreateContentType(resourceFullName.Name, p, relevantContentTypeUsage)))}'";

Reply:
otherwise postgres returns values in different order for assignedProfilesForRequest and throws error in postman test , I need to sorted the assignedProfilesForRequest to get same ordered values to have same error message in response

Reply:
So then why not just perform the `Sort` down in the `WriteForbiddenResponse` method in the case where you actually need it, instead of adding to the 100% execution path? Move it down to that method.

Reply:
I updated it . Thanks